frontend/model/state.js

```diff
@@ -51,6 +104,21 @@ const checkedUsername = (state: Object, username: string, userID: string) => {
   }
 }
 
+// Find the 'anyone can join' invite ID. Since there could be multiple, and some
+// of those could have exipred, we need a for loop
+const anyoneCanJoinInviteId = (invites: Object, getters: Object): ?string =>
+  Object.keys(invites).find(invite =>
+    // First, we want 'anyone can join' invites
+    invites[invite].creatorID === INVITE_INITIAL_CREATOR &&
+    // and that haven't been revoked
+    getters.currentGroupState._vm.invites[invite].status === INVITE_STATUS.VALID &&
+    // and that haven't expired (using negative logic because expires could be
+    // undefined for non expiring-invites)
+    !(getters.currentGroupState._vm.invites[invite].expires >= Date.now()) &&
```

Comment:
Calude is wrong.

`!(5 >= 5)` => false
`!(3 >= 5)` => true
`!(NaN >= 5)` => true

Reply:
Can you explain more?

If the expires is in the future from now, it means the invite hasn't expired yet. And yet, in that case, you flip the true to a false

Reply:
I see. I think you're right, and that it should be `<` instead of `>=`.

Background: it was meant to be the inverse of this:

```
const isWelcomeInviteExpired = this.currentWelcomeInvite.expires < Date.now()
```

I inverted the logic and then inverted it inside too, which I shouldn't have done.